tests.py

```diff
@@ -292,7 +299,10 @@ def test_load_ui_signals():
     """Tests to see if the baseinstance connects signals properly"""
     import sys
     from Qt import QtWidgets, QtCompat
-    app = QtWidgets.QApplication(sys.argv)
+    app = (
```

Comment:
Question about this actually; during tests, every test should be running in its own Python interpreter, so there *should* never be any need to look for an existing instance. Have you run into problems trying to instantiate a new application? That could be a bug.